scripts/validate_docstrings.py

```diff
@@ -328,6 +432,13 @@ def validate_one(func_name):
         errs.append('Private classes ({}) should not be mentioned in public '
                     'docstring.'.format(mentioned_errs))
 
+    if not doc.see_also:
+        errs.append('See Also section not found')
+    else:
+        for rel_name, rel_desc in doc.see_also:
```

Comment:
This gives me an error for ``scripts/validate_docstrings.py pandas.DataFrame.join`` (and others as well):

Traceback (most recent call last):
  File "scripts/validate_docstrings.py", line 483, in <module>
    sys.exit(main(args.function))
  File "scripts/validate_docstrings.py", line 469, in main
    return validate_one(function)
  File "scripts/validate_docstrings.py", line 438, in validate_one
    for rel_name, rel_desc in doc.see_also:
ValueError: too many values to unpack (expected 2)